### ISSUE TITLE
RETableViewOptionsController style

### ISSUE THREAD

**Issue by ericertl** [romaonthego/RETableViewManager#142]
Hi Roman.
I'm can't remove the header and footer shown in the RETableViewOptionsController. The original tableview doesn't have any headers (the section is built using RETableViewSection *section = [RETableViewSection section];) and the RETableViewOptionsController is using the same style (optionsController.style = section.style;)
Here's a screenshot of the options view controller:
![image](https://f.cloud.github.com/assets/777783/2262042/06882694-9e56-11e3-81aa-18e3307cdeb9.png)

**Comment by jbasttdi** [comment]
Yes,I have the same problem and want to remove to top space. HELP Please!

**Comment by preetigaur** [comment]
In the attribute inspector of your view controller, uncheck the "Adjust Scroll View Insets" option.

**Comment by ericertl** [comment]
I'll try this on monday
